radicle-cli/examples/rad-tag.md

```diff
@@ -9,12 +9,15 @@ We can now show the issue to check whether those tags were added:
 
 ```
 $ rad issue show 2e8c1bf3fe0532a314778357c886608a966a34bd
-title: flux capacitor underpowered
-state: open
-tags: [bug, good-first-issue]
-assignees: []
+╭─────────────────────────────────────────────────────────╮
+│ Title      flux capacitor underpowered                  │
+│ Assignees  No one                                       │
+│ Tags       bug, good-first-issue                        │
```

Comment:
Out of interest (and not to do with this PR at all), what happens when the list becomes really long? Would it be able to format itself onto the next line?

Reply:
I guess the TUI should implement the paginator API?

From the cli I think it is better to implement smart filtering? like from date range or somethings?

Reply:
No, I mean the `Tags` list :)

Reply:
Ya, maybe this is something we should catch in an issue and solve it in another PR :)